doc/source/rllib/rllib-rlmodule.rst

```diff
@@ -31,33 +36,60 @@ RL Modules (Alpha)
          - Fully-connected
          - Image inputs (CNN)
          - RNN support (LSTM)
-         - Complex observations (ComplexNet)
        * - **PPO**
          - |pytorch| |tensorflow|
          - |pytorch| |tensorflow|
          - |pytorch| |tensorflow|
          -
-         - |pytorch|
        * - **Impala**
          - |pytorch| |tensorflow|
          - |pytorch| |tensorflow|
          - |pytorch| |tensorflow|
          -
-         - |pytorch|
        * - **APPO**
          - |pytorch| |tensorflow|
          - |pytorch| |tensorflow|
          - |pytorch| |tensorflow|
-         - 
-         - 
+         -
+       * - **SAC**
+         -
+         - |pytorch|
+         -
+         -
+
 
+RL Module is a neural network container that implements three public methods:
+:py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.forward_train`,
+:py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.forward_exploration`,
+and :py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.forward_inference`.
+Each method corresponds to a distinct reinforcement learning and/or production phase.
 
+The :py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.forward_inference` method computes
+actions in a purely greedy fashion. It is normally used only in production settings (after
+training has been completed) or for evaluation purposes parallel to training.
+RLlib's :py:class:`~ray.rllib.evaluation.env_runner.EnvRunner` classes will chose this method
+if the `explore` flag in the :py:class:`~ray.rllib.algorithms.algorithm_config.AlgorithmConfig`
+is set to False.
 
-RL Module is a neural network container that implements three public methods: :py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.forward_train`, :py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.forward_exploration`, and :py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.forward_inference`. Each method corresponds to a distinct reinforcement learning phase.
+:py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.forward_exploration` computes actions
+in an exploratory way, balancing exploration and exploitation. It is the method normally used
+for training data collection by RLlib's :py:class:`~ray.rllib.evaluation.env_runner.EnvRunner` classes
+(if the `explore` flag in the :py:class:`~ray.rllib.algorithms.algorithm_config.AlgorithmConfig`
+is set to True, which it is by default).
+However, note that this method can also be used after training in an inference-only phase, if the user
+explicitly want the computed actions to be exploratory. Some environments - like the rock-paper-scissors game -
```

Comment:
Would put the last sentence in brackets "... (some environments ....)". It is not of direct interest at this point - it's an additional info for the reader asking for the "why".

Reply:
Probably adding something like "i.e. this implements a stochastic policy during inference".